packages/kit/src/runtime/server/utils.js

```diff
@@ -98,7 +98,8 @@ export async function handle_fatal_error(event, options, error) {
  */
 export async function handle_error_and_jsonify(event, options, error) {
 	if (error instanceof HttpError) {
-		return error.body;
+		// @ts-expect-error custom user errors may not have a message field if App.Error is overwritten
+		return { message: "Unknown Error", ...error.body };
```

Comment:
You should run `pnpm format`. I'm assuming it's failing because of the double quotes
```suggestion
		return { message: 'Unknown Error', ...error.body };
```